numpy/_core/code_generators/generate_umath.py

```diff
@@ -94,7 +94,8 @@ def finish_signature(self, nin, nout):
 
 
 def _check_order(types1, types2):
-    dtype_order = allP + "O"
+    # Insert kK (int64) after all other ints (assumes long long isn't larger)
```

Comment:
Can you add a sentence explaining why int64 needs to be last? It's not clear to me just reading this over.

Reply:
kK is assumed to be the largest int.  This function checks the order.  Let's put a sentence of doc to the function, I suspect that should solve this also.